synapse/python_dependencies.py

```diff
@@ -79,7 +79,7 @@
     # Fedora 31 only has 19.1, so if we want to upgrade we should wait until 33
     # is out in November.)
     # Note: 21.1.0 broke `/sync`, see #9936
-    "attrs>=19.1.0,<21.1.0",
+    "attrs>=19.1.0,!=21.1.0",
```

Comment:
Probably unnecessary; 21.1.0 was yanked from PyPI, so it effectively no longer exists. We can go back to just `attrs>=19.1.0`

Reply:
Mmm, true. Though I think since we know its bad we may as well leave it in there. I'm not sure if e.g. pypi mirrors and the like handle yanking etc.